src/wfs/src/test/java/org/geoserver/wfs/v1_1/DescribeFeatureTypeTest.java

```diff
@@ -299,4 +305,24 @@ public void testCustomSchema() throws Exception {
 //            future.get();
 //        }
 //    }
+
+    /**
+     * Tests that WFS schema is not imported in a DescribeFeatureType response.
+     *
+     * @throws Exception
+     */
+    @Test
+    public void testNoWfsSchemaImport() throws Exception {
```

Comment:
WFS 1.0 ``DescribeFeatureType`` operation was not affected by this fix right, i.e. WFS schema was already not included ? Thanks for taking the time to add this test, this helps indeed keeping the behavior consistent.

Reply:
Right, it was not affected, but I figured one more test would not kill anybody... ;-)